src/register_axis.cpp

```diff
@@ -156,15 +156,15 @@ void register_axes(py::module &mod) {
 
     register_axis_sub_types<axis::category_int, axis::category_int_growth>(
         mod,
-        {"_category_int", "_category_int_growth"},
+        {{"_category_int", "_category_int_growth"}},
         "Axis with discontiguous integer bins",
         py::init<std::vector<int>, metadata_t>(),
         "categories"_a,
         "metadata"_a = py::none());
 
     register_axis_sub_types<axis::category_str, axis::category_str_growth>(
         mod,
-        {"_category_str", "_category_str_growth"},
+        {{"_category_str", "_category_str_growth"}},
```

Comment:
Doesn't matter, but why are these now double bracketed?

Reply:
clang-5 was complaining about it.